tests/unit/challenges/test_views.py

```diff
@@ -486,6 +486,17 @@ def test_particular_challenge_for_mapping_with_participant_team_does_not_exist(s
         self.assertEqual(response.data, expected)
         self.assertEqual(response.status_code, status.HTTP_406_NOT_ACCEPTABLE)
 
+    def test_particular_challenge_for_mapping_with_past_challenge(self):
+        self.challenge2.end_date = timezone.now() - timedelta(days=1)
```

Comment:
Please change the name to `test_participant_team_for_mapping_with_past_challenge`